constrained-voucher.mkd

```diff
@@ -236,13 +244,54 @@ and ct=TBD3 stands for Content-Format "application/voucher-cose+cbor".
 
 Content-Formats TBD2 and TBD3 are defined in this document.
 
-The Content-Format ("application/json") 50 MAY be supported.
-Content-Formats ("application/cbor") 60, TBD2, and TBD3 MUST be supported by the Registrar.
+The Content-Format ("application/json") 50 MAY be supported and 60 MUST be supported by the Registrar for the /vs and /es resources.
+Content-Format TBD3 MUST be supported by the Registrar for the /rv resource. If the "ct" attribute is not indicated for this resource, this implies that at least TBD3 is supported.
 
 The Pledge and MASA need to support one or more formats for the voucher.
 The MASA needds to support whatever formats that the pledge's produced by that manufacturer supports.
 
-# BRSKI-MASA protocol {#brski-masa}
+## Extensions to BRSKI {#brski-extensions}
+
+A Pledge that only supports the EST-coaps enrollment method SHOULD NOT use discovery for BRSKI resources, since it is more efficient to just try
+the supported enrollment method via the well-known BRSKI/EST-coaps resources, and it avoids the Pledge having to do complex CoRE Link Format parsing.
+A Registrar SHOULD host any discoverable BRSKI resources on the same (UDP) server port that the Pledge's DTLS connection is using. 
```

Comment:
If we say SHOULD here instead of MUST, is there an exception?  Otherwise, let's say MUST.